bin/eg-generator.js

```diff
@@ -46,15 +46,14 @@ module.exports = class EgGenerator extends Generator {
   // configuration defaults
   _wrapConfig (yargs) {
     return yargs
-      .boolean(['no-color', 'q', 'v'])
```

Comment:
Didn't like `--no-color`?  This is mostly for anything using Yeoman output.

Reply:
Ah — that's where's used! I've been looking around and I wasn't able to find it.

Let me verify one more time then. Hold on on this please 🙏